CHANGELOG.md

```diff
@@ -32,6 +32,21 @@ Deprecated features
 Deprecated features still exist and therefore existing code should still work
 but they may be removed in some future release of the library.
 
+* Relations over data have been moved from the `Relation` subtree to the `Data`
+  subtree. The full list of moves is as follows:
+  - `Relation.Binary.List.Pointwise`       ↦ `Data.List.Relations.Pointwise`
```

Comment:
Minor bikeshedding:  Why not `Relation` instead of `Relations`?  I think in general we use singular rather than plural.